package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "devDependencies": {
+    "@commitlint/cli": "^17.7.2",
+    "@commitlint/config-conventional": "^17.7.0",
+    "husky": "^8.0.3"
+  }
+}
```

Comment:
I think we need to add a `prepare` script as well. To install the pre-commit hook with `npm install` command.

```json
{
....
"scripts": {
    ....
    "prepare": "husky install",
   ....
}
....
}
```

Reply:
Yes, I shall make the changes. Is there anything else that needs to be added?

Reply:
> Yes, I shall make the changes. Is there anything else that needs to be added?

For this issue, I think it's fine